src/svgs/index.ts

```diff
@@ -1,3 +1,4 @@
+export * from "./CouponBody";
 export * from "./RestaurantDescriptionCoupon";
 export * from "./RestaurantDescriptionCutlery";
 export * from "./RestaurantDescriptionLocation";
```

Comment:
move svgs to base/component folder.
// index.tsx
`export { default as TempTwo } from "./TempTwo";
`

//SVG.props.ts
`export type SVGProps = {
  fill?: string;
  width?: number;
  height?: number;
  stroke?: string;
};
`
<img width="210" alt="image" src="https://github.com/langara-capstone-Spring2024/nom-fe-revamp/assets/54840932/9fac411b-8150-437d-9a74-28afcf7f30fa">